src/V3AstNodeMath.h

```diff
@@ -0,0 +1,4613 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: AstNode subclasses representing expressions
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2022 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#ifndef VERILATOR_V3ASTNODEMATH_H_
+#define VERILATOR_V3ASTNODEMATH_H_
+
+#ifndef VERILATOR_V3AST_H_
+#error "Use V3Ast.h as the include"
+#include "V3Ast.h"  // This helps code analysis tools pick up symbols in V3Ast.h
+#ifndef VL_NOT_FINAL
+#define VL_NOT_FINAL
+#endif
```

Comment:
Have just in V3Ast.h?

Reply:
This us some cruft to get my slightly broken IDE to work nicely that crept in with the noise, will remove or minimze.

Reply:
If there's a need that's fine, just comment it's for an IDE so someone doesn't forget and rip it out.